cli/main.rs

```diff
@@ -51,11 +51,17 @@ fn main() {
 }
 
 fn repl(mut jstime: jstime::JSTime) {
+    use dirs::home_dir;
     use rustyline::{error::ReadlineError, Editor};
 
+    let mut history_path = home_dir().unwrap();
```

Comment:
`home_dir()` can return a `None` if `$HOME` (or equiv) isn't set. What do you want the failure mode to be here? Maybe a better approach if there's no place to store the history file is to not use a history file?

Reply:
Oh I this was seen by snek at same time.

Reply:
I'm totally open to better handling this just unsure the appropriate way to do this in rust. Wanna push a commit?

Reply:
Commit pushed!